app/assets/stylesheets/application.scss

```diff
@@ -177,13 +177,18 @@ body {
       }
     }
 
+    .diy-header {
+      max-width: 450px;
+    }
+
     .diy-subheader {
       font-weight: 200;
-      max-width: 391px;
+      max-width: 450px;
     }
 
     .grid {
       position: relative;
+      min-height: 355px;
```

Comment:
ensures the hero image isn't cut off at the bottom with short copy (the current live version is short)

Reply:
good catch!